### PR TITLE
docs(agent): Add agent configuration with vault

### PR DESCRIPTION
**Pull request by nhtzr**

### CODE REVIEW

File: content/en/docs/armory-agent/agent-vault.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+{{< prism lang="yaml" line="10-11" >}}
+# ./kustomization.yaml
+# Pre-existing SpinnakerService resource (may be different)
+namespace: spinnaker
+resources:
+  - spinnakerservice.yaml
+bases:
+# Armory agent deployment
+  - https://armory.jfrog.io/artifactory/manifests/kubesvc/armory-agent-0.5.11-kustomize.tar.gz
```
Comment (german-muzquiz) [inline content/en/docs/armory-agent/agent-vault.md]: Do http references like this actually work? @th3morg was not able to make it work with recent versions of kustomize

Comment (nhtzr) [reply content/en/docs/armory-agent/agent-vault.md]: It works in kustomize 3.6.1 
I just noticed that kustomize 4.1.6 fails with `Error: accumulating resources: accumulation err='accumulating resources from 'https://armory.jfrog.io/artifactory/manifests/kubesvc/armory-agent-0.5.11-kustomize.tar.gz': yaml: control characters are not allowed': git cmd = '/usr/local/bin/git fetch --depth=1 origin HEAD': exit status 128`
kubectl kustomize does fail with `Error: rawResources failed to read`

Comment (nhtzr) [reply content/en/docs/armory-agent/agent-vault.md]: I'll update both

Comment (nhtzr) [reply content/en/docs/armory-agent/agent-vault.md]: updated

File: content/en/docs/armory-agent/armory-agent-quick.md [hunk 2]
```diff
@@ -152,82 +145,115 @@ If you are not using Kustomize, you can still use the same manifests.
...
   - name: account01
-    # /kubeconfigfiles/ is the path to the config files
-    # as mounted from the `kubeconfigs-secret` Kubernetes secret
     kubeconfigFile: /kubeconfigfiles/kubecfg-account01.yaml
-    ...
-  - ...
-...
 ```
 
-* For installations without gRPC TLS connections, you should include `clouddriver.insecure: true` in the Agent options.
```
Comment (german-muzquiz) [inline content/en/docs/armory-agent/armory-agent-quick.md:208]: Was this info (and line below) moved somewhere else?

Comment (nhtzr) [reply content/en/docs/armory-agent/armory-agent-quick.md:208]: yup, to [line 160](https://github.com/armory/docs/pull/671/files#diff-d51fcd92337deee3538163f79a916f3fe109ba4dd7ec6966019c375466dafbf3R160)

**Review by german-muzquiz — APPROVED** [review]

**Pull request merged**